charts/incubator/tt-rss/values.yaml

```diff
@@ -0,0 +1,179 @@
+image:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+nginxImage:
+  repository: cthulhoo/ttrss-web-nginx
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f963b38ef8695d05c7f219e9405648dcb2f1f3c0b81609353bb1ba078bc8e318
+updaterImage:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+persistence:
+  html:
+      enabled: true
+      targetSelector:
+        main:
+          main:
+            mountPath: /var/www/html
+        nginx:
+          nginx:
+            mountPath: /var/www/html
+            readOnly: true
+        updater:
+          updater:
+            mountPath: /var/www/html
+  config:
+    enabled: true
+    targetSelector:
+        main:
+          main:
+            mountPath: /opt/tt-rss/config.d
+        updater:
+          updater:
+            mountPath: /opt/tt-rss/config.d
+  # plugins:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/plugins.local
+  # themes:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/themes.local
```

Comment:
What's up with those?

Reply:
Left over from the previous implementation. tt-rss now supports installing plugins from a dialog, but I still have a to-do set to figure out how to allow users to manually upload themes to persistent storage.

Reply:
If you install the plugin from the dialog, where does it gets placed in the filesystem?

Reply:
Plugins installed from the dialog are automatically placed in `/var/www/html/tt-rss/plugins.local`. Which means they should be under the html persistent storage.

There's no dialog for themes, which should manually be placed in `/var/www/html/tt-rss/themes.local`. If it's possible to separate this path from the html storage, then users could specify for it to be a separate NFS/etc mount to allow adding themes easily.

Reply:
You can enable the commented out persistence above, 
should be enough to separate those